packages/istanbul-report-modern-html/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "istanbul-report-modern-html",
+  "version": "1.0.0",
+  "description": "istanbul report - modern html",
+  "author": "Luke Page",
+  "main": "index.js",
+  "files": [
+    "index.js",
+    "lib",
+    "assets"
+  ],
+  "scripts": {
+    "test-disabled": "mocha --recursive",
```

Comment:
This should be made compatible with the other packages/* test scripts so coverage is collected :)

Reply:
yep, this is the last tick mark - add tests. If I leave as is without tests then it causes the build to fail